backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/database/Schema.kt

```diff
@@ -27,10 +27,14 @@ class CardEntity(id: EntityID<Int>) : IntEntity(id) {
     var revoked by Cards.revoked
     var cardDetailsHash by Cards.cardDetailsHash
 
-    var expirationDate: LocalDateTime
-        get() = LocalDateTime.ofEpochSecond(expirationDateEpochSeconds, 0, ZoneOffset.UTC)
+    var expirationDate: LocalDateTime?
+        get() = if (expirationDateEpochSeconds > 0) LocalDateTime.ofEpochSecond(
+            expirationDateEpochSeconds,
+            0,
+            ZoneOffset.UTC
+        ) else null
```

Comment:
I think we should rather make `Cards.expirationDate` nullable in the database, otherwise this seems pretty hacky.
Also I think it would be better, if we had expirationDate as a nullable already in GraphQL to make it explicit...